app/assets/stylesheets/views/_homepage.scss

```diff
@@ -216,108 +216,134 @@ body.homepage {
       }
     }
     form#header-search {
-      padding-bottom: 30px;
+      @extend %contain-floats;
+      margin-bottom: 30px;
 
-      .header-search-content {
-        @extend %contain-floats;
-        position: relative;
-        background: #fff;
-      }
       label {
-        float: left;
-        height: 40px;
-        line-height: 40px;
-        text-indent: 15px;
-        overflow: hidden;
-        display: block;
-        margin-right: 10px;
-        color: $secondary-text-colour;
+        color: $white;
```

Comment:
Minor: I don't think you need this as all text inside the header is already white. The label would just inherit that.